### PR TITLE
Dinosaur catalog - iteration 3

### PR DESCRIPTION
**Pull request by jcurlee**

### CODE REVIEW

File: dino_catalog/dinosaur_catalog.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    current_catalog.select { |row| row[key] && value.casecmp(row[key]) == 0 }
+  end
+
+  private
+
+  def find_by_weight(size)
+    @dinosaur_catalog.select do |row|
+      weight = row[:weight].to_i
+
+      size == 'large' ? weight > TWO_TONS : weight <= TWO_TONS
```
Comment (oharace) [inline dino_catalog/dinosaur_catalog.rb]: Nice job extracting two tons to a constant.

File: dino_catalog/dinodex_ui.rb [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    while @running
+      show_menu_options
+      handle_user_input
+    end
+  end
+
+  def self.handle_user_input
+    user_input = collect_user_input
+
+    if %w(b j q).include? user_input
```
Comment (oharace) [inline dino_catalog/dinodex_ui.rb]: Wrap arguments in parentheses -> `if %w(b j q).include?(user_input)`

**Comment by oharace** [comment]
Excellent third iteration! :+1:

**Comment by paulghaddad** [comment]
@jcurlee Nice work! You're free to move on!

**Pull request closed without merging**